pkg/api/types.go

```diff
@@ -607,6 +607,10 @@ type SecretVolumeSource struct {
 	// the volume setup will error. Paths must be relative and may not contain
 	// the '..' path or start with '..'.
 	Items []KeyToPath `json:"items,omitempty"`
+	// Mode bits to use on created files by default. The used mode bits will
+	// be the provided AND 0777.
+	// Directories within the path are not affected by this setting
+	DefaultMode int32 `json:"defaultMode,omitempty"`
```

Comment:
Oh, didn't know this convention. Will change it. Thanks! Just curious, is there some doc I can check for these conventions? And maybe too curious, but why a pointer? Because nil can be used as "do the default behavior" ?